### PR TITLE
[9.0][IMP] stock_deposit: Set partner equal to owner for deposits pickings

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Check Travis status

File: stock_deposit/models/stock_quant.py [hunk 1]
```diff
@@ -22,3 +22,21 @@ def quants_get(self, qty, move, ops=False, domain=None,
             domain = self._block_deposit_quants(domain)
         return super(StockQuant, self).quants_get(
             qty, move, ops, domain, removal_strategy)
+
+    @api.multi
+    def write(self, vals):
+        if 'reservation_id' in vals:
```
Comment (pedrobaeza) [inline stock_deposit/models/stock_quant.py:28]: You should do this in quant_reserve method

Comment (sergio-teruel) [reply stock_deposit/models/stock_quant.py:28]: Yes you are rigth... But this method is not inheritable....

Comment (pedrobaeza) [reply stock_deposit/models/stock_quant.py:28]: OK, I see that there's no other possibility.

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**